pages/docs/user-docs/build-docker-module.md

```diff
@@ -0,0 +1,43 @@
+---
+title: docker bootstrap module
+sidebar: user_docs
+permalink: build-docker-module
+folder: docs
+toc: false
+---
+
+This module allows you to build a container from an existing container hosted on Docker Hub or another Docker registry. 
+
+{% include toc.html %}
+
+## Overview
+You can use an existing container image as your "base," and then add customization. For example, instead of using the `debootstrap` module to start an Ubuntu container, you might use the official Ubuntu container from Docker Hub as a base.  You could do the same with CentOS, Debian, Arch, Suse, Alpine, BusyBox, etc.  Or maybe you want to build a container that uses CUDA and cuDNN to leverage the GPU, but ou don't want to install from scratch.  You can start with one of the `nvidia/cuda` containers and install your software on top of that.
+
+## Keywords
```

Comment:
This should do the same as above, and talk about Docker with regard to layers.Eg:

Docker images are comprised of layers that are assembled at runtime to create an image. Singularity can use these same layers to formulate a base image, onto which you can add your own custom software.  For example, you might use Docker's Ubuntu image layers to create an Ubuntu Singularity container. You might already have many scientific Docker containers that you can seamlessly convert to Singularity. 

and then a bug - "but you don't want to install from scratch"